features/duck-player.json

```diff
@@ -6,7 +6,7 @@
     "exceptions": [],
     "features": {
         "pip": {
-            "state": "disabled"
+            "state": "enabled"
```

Comment:
Can you keep this disabled in case other platforms are implemented?

Reply:
Ok I Updated the code, thank you!